KerbalStuff/email.py

```diff
@@ -33,23 +33,25 @@ def send_password_reset(user):
 def send_password_changed(user):
     with open("emails/password-changed") as f:
         message = html.unescape(
-            chevron.render(f.read(), {'user': user, 'site_name': _cfg('site-name'), "domain": _cfg("domain")}))
+            chevron.render(f.read(), {
+                'user': user,
+                'site_name': _cfg('site-name'),
+                "domain": _cfg("domain"),
+                'support_channels': support_channels_to_map()
```

Comment:
From what I can tell, we should be able to pass `_cfgd('support-channels')` directly to `render` and access the keys and values in the template. Do we need to generate this intermediate representation as a list of dicts?

Reply:
Pystache/Chevron needs keys it can access when looping over a list:
```
{{#support_channels}}
  * {{name}}: {{channel_url}}
{{/support_channels}}
```
It would be cool if we simply could do something like `$key` and `$value` to get the name of the key and the one of the corresponding value, but it wasn't possible in pystache, and I suspect it's also not possible in chevron.

Reply:
Yeah I had in mind this from jinja2:

```jinja2
{% for key, value in support_channels.items() %}
    * {{key}}: {{value}}
{% endfor %}
```

... but it does seem that Chevron can't do it. Never mind!